t/cli-plugins/validateonly.py

```diff
@@ -0,0 +1,42 @@
+from typing import Mapping
+
+from flux.cli.plugin import CLIPlugin
+
+
+class ValidateShellVerboseOpt(CLIPlugin):
+    name = "shell.options.verbose"
+
+    def validate(self, jobspec):
+        try:
+            verbosity = jobspec.attributes["system"]["shell"]["options"]["verbose"]
+            if not verbosity.isdigit():
+                raise ValueError(f"{self.name}: expected integer, got {verbosity}")
```

Comment:
IMHO it could be useful to return the _type_ of `verbosity` instead of `verbosity` itself in this exception message, but you could probably take or leave this.

Reply:
Hmm, this plugin isn't being tested _at all_, that seems bad.